src/HavenoDaemon.ts

```diff
@@ -311,6 +330,148 @@ class HavenoDaemon {
       });
     });
   }
+
+  ///////////////////////////////////////////////
+  // Account
+  ///////////////////////////////////////////////
+
+  /**
+   * Determine if an Account Exists.
+   * 
+   * @return {boolean} whether an account exists
+   */
+  async accountExists(): Promise<boolean> {
+    let that = this;
+    return new Promise(function(resolve, reject) {
+      that._accountClient.accountExists(new AccountExistsRequest(), {password: that._password}, function(err: grpcWeb.RpcError, response: AccountExistsReply) {
+        if (err) reject(err);
+        else resolve(response.getAccountExists());
+      });
+    });
+  }
+
+  /**
+   * Determine if an Account is open.
+   * 
+   * @return {boolean} whether an account is open
+   */
+  async isAccountOpen(): Promise<boolean> {
+    let that = this;
+    return new Promise(function(resolve, reject) {
+      that._accountClient.isAccountOpen(new IsAccountOpenRequest(), {password: that._password}, function(err: grpcWeb.RpcError, response: IsAccountOpenReply) {
+        if (err) reject(err);
+        else resolve(response.getIsAccountOpen());
+      });
+    });
+  }
+
+  /**
+   * Create an Account.
+   * 
+   * @param {string} password - the password for the Account
+   */
+  async createAccount(password: string): Promise<void> {
+    let that = this;
+    return new Promise(function(resolve, reject) {
+      that._accountClient.createAccount(new CreateAccountRequest().setPassword(password), {password: that._password}, function(err: grpcWeb.RpcError, response: CreateAccountReply) {
+        if (err) reject(err);
+        else resolve();
+      });
+    });
+  }
+
+  /**
+   * Create an Account.
+   * 
+   * @param {string} password - the password for the Account
+   */
+  async openAccount(password: string): Promise<void> {
+    let that = this;
+    return new Promise(function(resolve, reject) {
+      that._accountClient.openAccount(new OpenAccountRequest().setPassword(password), {password: that._password}, function(err: grpcWeb.RpcError, response: OpenAccountReply) {
+        if (err) reject(err);
+        else resolve();
+      });
+    });
+  }
+
+  /**
+   * Close an Account.
+   * 
+   */
+  async closeAccount(): Promise<void> {
+    let that = this;
+    return new Promise(function(resolve, reject) {
+      that._accountClient.closeAccount(new CloseAccountRequest(), {password: that._password}, function(err: grpcWeb.RpcError, response: CloseAccountReply) {
+        if (err) reject(err);
+        else resolve();
+      });
+    });
+  }
+
+  async backupAccount(): Promise<Buffer> {
```

Comment:
Should return `Attachment` with file name and bytes.

Reply:
changed